astro/src/components/download/DownloadWidget.astro

```diff
@@ -0,0 +1,624 @@
+      <li class="group"><a data-tab-button href="#docker-vanilla" class="block border-b-2 border-transparent group-[.active]:border-indigo-400 flex-grow px-7 py-3 text-md text-slate-400 group-[.active]:text-white">Docker Vanilla</a></li>
+    </ul>
+    <hr class="border-slate-700"/>
+    <div data-tab-content id="docker-compose" class="m-8" data-reset={step = 1}>
+      <h2 class="mb-4 text-2xl">Docker Compose</h2>
+      <p class="mb-6">
+        Use the following Docker Compose commands to get up and running quickly with FusionAuth, PostgreSQL, and Elasticsearch. You might need to modify it if you are using a different operating system. Feel free to open an issue in <a href="https://github.com/FusionAuth/fusionauth-issues" class={anchorClass}>GitHub</a> if you find any issues.
+      </p>
+      <p class="mb-4">
+        <strong>Step { step++ }:</strong> Download the Docker files
+      </p>
+      <ul class="flex mb-4">
+        <li class="group active"><a data-tab-button href="#docker-compose-download-nix" class="block border-b-2 border-transparent font-bold px-7 py-3 text-xs text-slate-400 group-[.active]:border-indigo-400 group-[.active]:text-white">Linux/Unix/macOS</a></li>
+        <li class="group"><a data-tab-button href="#docker-compose-download-windows" class="block border-b-2 border-transparent flex-grow px-7 py-3 text-xs text-slate-400 group-[.active]:border-indigo-400 group-[.active]:text-white">Windows</a></li>
+      </ul>
+      <div data-tab-content id="docker-compose-download-nix">
+        <Code>
+          curl -fsSL https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/main/docker/fusionauth/docker-compose.yml > docker-compose.yml && \
+          curl -fsSL https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/main/docker/fusionauth/.env > .env
+        </Code>
+      </div>
+      <div data-tab-content id="docker-compose-download-windows" class="hidden">
+        <Code>
+          Invoke-WebRequest -UseBasicParsing https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/main/docker/fusionauth/docker-compose.yml -OutFile docker-compose.yml
+          Invoke-WebRequest -UseBasicParsing https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/main/docker/fusionauth/.env -OutFile .env
+        </Code>
+      </div>
+      { kickstartEnabled &&
+          <p class="mb-2 mt-12">
+            <strong>Step { step++ }:</strong> Download the Kickstart files needed for this project
+          </p>
+          <p class="mb-4">
+            <em class="text-xs">NOTE: The Docker Compose configuration loads these Kickstart files automatically.</em>
+          </p>
+          <Kickstart env={false} includeNix={true} includeWindows={true} name={kickstartName} section="docker-compose-kickstart"/>
+      }
+      <p class="mb-6 mt-12">
+        <strong>Step { step++ }:</strong> Start the Docker Compose containers
+      </p>
+      <Code>
+        docker-compose up
```

Comment:
```suggestion
        docker compose up
```

I think most of our newer stuff uses `docker compose` instead of `docker-compose`. I thought the latter was deprecated.